src/plugins/index.js

```diff
@@ -77,12 +77,13 @@ export default class Plugins {
 
   async findCommand (cmd: string): Promise<?Class<Command<*>>> {
     for (let plugin of this.plugins) {
+      let nsCmd
       if (plugin.namespace) {
         let split = cmd.split(':')
         if (plugin.namespace !== split[0]) return
-        cmd = split.slice(1, split.length).join(':')
+        nsCmd = split.slice(1, split.length).join(':')
       }
-      let c = await plugin.findCommand(cmd)
+      let c = await plugin.findCommand(nsCmd || cmd)
```

Comment:
If we want this behavior, it will get us `sfdx force` where force can then be an cmd alias or id. If not there is still a bug fix here that needs to get in.